civic-tech-club/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Step 1: Base Stage for both Dev and Prod
+FROM node:16 AS base
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and install dependencies
+COPY package*.json ./
+RUN npm install
+
+# Install Sass globally
+RUN npm install -g sass
+
+# Install npm-run-all globally for running multiple scripts in parallel
+RUN npm install -g npm-run-all
```

Comment:
I would highly recommend versioning all dependencies in package.json and adding relevant scripts there. This keeps us open to modifying how things are run or installed without needing to update more than one file.

Reply:
I think this is done, partially because of starting from scratch with vite

Reply:
siiick